pkg/go.mod

```diff
@@ -96,10 +96,10 @@ require (
 	github.com/segmentio/encoding v0.3.5
 	github.com/shirou/gopsutil/v3 v3.22.3
 	github.com/spf13/afero v1.9.5
-	go.opentelemetry.io/otel v1.7.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.7.0
-	go.opentelemetry.io/otel/sdk v1.7.0
-	go.opentelemetry.io/otel/trace v1.7.0
+	go.opentelemetry.io/otel v1.19.0
```

Comment:
I am not sure what's going on here, but I got a message

```
used metadata's version v1.19.0 rather than go.mod's version v1.7.0
```

I guess something else is forcing the minimum version up to v1.19.0? This is also probably why I got the nil pointer exception.

Reply:
Yeah @PollRobots had seen this as well. I have no idea how y'all are ending up with `v1.19.0` and I am seeing `v1.7.0`. Happy to go with this if it works.